### PR TITLE
feat(content,port): Centipede overhaul

### PR DESCRIPTION
**Pull request by Zlorthishen**
## Purpose of change
ports in the following dda pull requests
- https://github.com/CleverRaven/Cataclysm-DDA/pull/45868 by Venera3
- https://github.com/CleverRaven/Cataclysm-DDA/pull/45416 by Venera3

## Describe the solution
adds several new varieties of centipedes, as well as removing an obsolete variety
creates a monster_faction and species for centipedes
also ports over elements of Diversified Venoms too

### CODE REVIEW

**Review by scarf005 — DISMISSED** [review]
please add co-authors to each commits.

**Comment by Zlorthishen** [comment]
> please add co-authors to each commits.

image

**Comment by scarf005** [comment]
sorry @Zlorthishen, i didn't know that github mobile lacked commit details/co-author display like on browser.

File: data/json/effects.json [hunk 1]
```diff
@@ -493,6 +493,68 @@
...
   },
+  {
+    "id": "venom_dmg",
+    "type": "effect_type",
+    "name": [ "Poisoned Wound" ],
+    "desc": [ "The wound swells visibly and burns like hell." ],
+    "apply_message": "Ow, that stings!",
+    "remove_message": "The burning pain subsides, but the damage is done.",
+    "rating": "bad",
+    "resist_traits": [ "POISRESIST" ],
```
Comment (chaosvolt) [inline data/json/effects.json:504]:
```suggestion
    "resist_traits": [ "POISRESIST" ],
    "resist_effects": [ "antitoxin" ],
```

File: data/json/effects.json [hunk 2]
```diff
@@ -493,6 +493,68 @@
...
+  },
+  {
+    "id": "venom_weaken",
+    "type": "effect_type",
+    "name": [ "Weakening Poison" ],
+    "desc": [ "You feel feeble." ],
+    "apply_message": "The sting itself barely hurt, but suddenly you feel exhausted.",
+    "remove_message": "The weight on your chest lessens.",
+    "rating": "bad",
+    "resist_traits": [ "POISRESIST" ],
```
Comment (chaosvolt) [inline data/json/effects.json:533]:
```suggestion
    "resist_traits": [ "POISRESIST" ],
    "resist_effects": [ "antitoxin" ],
```

**Review by chaosvolt — CHANGES_REQUESTED** [review]

**Review by scarf005 — APPROVED** [review]

**Pull request merged**